src/main/resources/rules.json

```diff
@@ -195,6 +195,98 @@
       "section": "Directory Structure",            
       "accept": "src/test/java/${connector_package}/automation/system/$",
       "assert": "src/test/java/${connector_package}/automation/system"   
-    }          
+    },
+    
+    
+    
+    {
+      "type": "source.pom",
+      "id": "mule_dependencies_provided",
+      "severity": "critical",
+      "brief": "Mule dependencies should be <scope>provided</scope>",
+      "description": "Mule dependencies should be <scope>provided</scope>",
+      "section": "Develop Best Practices",
+      "accept": "/pom:project/pom:dependencies/pom:dependency/pom:groupId[text()='org.mule.modules']",
+      "assert": "/pom:project/pom:dependencies/pom:dependency/pom:scope[text()='provided']"
```

Comment:
Hum... I'm not sure this is doing what it's meant to do... Let's say you have 2 dependencies for groupId 'org.mule.modules'. One has provided scope and the other doesn't... this assertion will let it pass, and it shouldn't.
Not very proficient with XPath, not sure how this could be achieved.
@veigap?